doc/installation/ubuntu.rst

```diff
@@ -24,17 +24,7 @@ Before installing privacyIDEA 3.0 or upgrading to 3.0 you need to add the reposi
 Add repository
 ~~~~~~~~~~~~~~
 
-Create a new file ``/etc/apt/sources.list.d/privacyidea-community.list`` with the
-following contents:
-
-   deb http://lancelot.netknights.it/community/xenial/stable xenial main
-
-"xenial" is for Ubuntu 16.04LTS. If you are running Ubuntu 18.04LTS, 
-replase "xenial" with "bionic":
-
-   deb http://lancelot.netknights.it/community/bionic/stable bionic main
-
-Download the signing key::
+The packages are digitially signed. First you need to download the signing key::
```

Comment:
*digitally

Reply:
done in a0089735 - you are welcome to squash the two commits.